syshook_utils.c

```diff
@@ -0,0 +1,64 @@
+struct sys_hook {
+    void** orig_func_pointer;
+    void* hook_func;
+    int sys_entry;
+};
+
+int set_page_rw(unsigned long addr)
+{
+    unsigned int level;
+    pte_t *pte = lookup_address(addr, &level);
+    if (pte->pte &~ _PAGE_RW) pte->pte |= _PAGE_RW;
+    return 0;
```

Comment:
seems like this function always succeeds, so change it to `void set_page_rw...`

Same goes with `set_page_ro` function

Reply:
done